1주차/셀프넘버.py

```diff
@@ -0,0 +1,18 @@
+def selfNumber(n):
+    my_list = []
+    number = str(n)
+    for i in number:
+        my_list.append(int(i))
+    return n + sum(my_list)
```

Comment:
```suggestion
    return n + sum(map(int, str(n)))
```
map 객체도 sum함수를 사용해서 내용물을 전부 더할 수 있어요